css/spinkit.css

```diff
@@ -42,8 +42,8 @@
  *  Usage:
  *
  *    <div class="sk-spinner sk-spinner-double-bounce">
- *      <div class="sk-double-bounce1"></div>
- *      <div class="sk-double-bounce2"></div>
+ *      <div class="sk-child sk-double-bounce1"></div>
+ *      <div class="sk-child sk-double-bounce2"></div>
```

Comment:
@jmagnusson Adding specific classes for shared styles here, instead of relying on for example `div`, :+1: / :-1: ?